src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -91,7 +91,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                 into dockerDir
             }
 
-            List<String> buildCommandLine = ['docker', 'build']
+            List<String> buildCommandLine = ['docker', 'build', '--quiet']
```

Comment:
@uschi2000 curious for your thoughts here, but builds for some of our projects are generating huge reports -- think we can eliminate a lot of junk build content by setting --quiet here.